### PR TITLE
[zh-cn] minor update task/administer-cluster

### PR DESCRIPTION
**Pull request by tico88612**
- Update `EncryptionConfiguration configuration API (v1)` link
- Add `task-tutorial-prereqs-node-upgrade.md` translate & replace
- `feature-state` replace `feature_gate_name` with `state` & `for_k8s_version`

### CODE REVIEW

File: content/zh-cn/includes/task-tutorial-prereqs-node-upgrade.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<!--
+You need to have shell access to all the nodes, and the kubectl command-line tool must
+be configured to communicate with your cluster. It is recommended to run this tutorial 
+on a cluster with at least two nodes that are not acting as control plane hosts.
+-->
+你必须拥有一个 Kubernetes 的集群，同时你必须配置 kubectl 命令行工具与你的集群通信。
```
Comment (windsonsea) [inline content/zh-cn/includes/task-tutorial-prereqs-node-upgrade.md:6]:
```suggestion
你必须有 Shell 能访问所有节点，且必须配置 kubectl 命令行工具让其与你的集群通信。
```

File: content/zh-cn/includes/task-tutorial-prereqs-node-upgrade.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+建议在至少有两个不作为控制平面主机的节点的集群上运行本教程。
```
Comment (windsonsea) [inline content/zh-cn/includes/task-tutorial-prereqs-node-upgrade.md:7]:
```suggestion
建议运行本教程的集群至少有两个节点，且这两个节点不能作为控制平面主机。
```

**Review by windsonsea — COMMENTED** [review]
Make Chinese text more fluent.

File: content/zh-cn/includes/task-tutorial-prereqs-node-upgrade.md (hunk 1 above)
Comment (tico88612) [reply content/zh-cn/includes/task-tutorial-prereqs-node-upgrade.md:6]: This is the same translation as `content/zh-cn/includes/task-tutorial-prereqs.md`.

Do you think it needs to be changed together?

Comment (windsonsea) [reply content/zh-cn/includes/task-tutorial-prereqs-node-upgrade.md:6]: Seems yes, the translation has some gap to the original english.

Comment (tico88612) [reply content/zh-cn/includes/task-tutorial-prereqs-node-upgrade.md:6]: Oh, I misread it. The two files are different.

**Comment by windsonsea** [comment]
It's required to squash your commits into a single one.

**Review by windsonsea — COMMENTED** [review]
Thanks
/approve

**Pull request merged**